### PR TITLE
Add option to install nightly version

### PR DESCRIPTION
**Pull request by iggy**
The nightly version (i.e. the next aptly release) will have an option to allow
better functionality when run from within salt (-batch mode). Use this nightly
build and alter the formula functionality to match.

### CODE REVIEW

File: aptly/init.sls [hunk 1]
```diff
@@ -10,13 +15,13 @@ aptly_repo:
       - pkg: aptly
 
 aptly:
-  pkg.installed:
+  pkg.latest:
     - name: aptly
```
Comment (whiteinge) [inline aptly/init.sls]: This should probably be config-gated in Pillar. Maybe `aptly:always_upgrade` with a `False` default. What do you think?

Comment (iggy) [inline aptly/init.sls]: Yeah, forrest and I were talking about it in irc. Probably just use the install_nightly key for this too as I doubt anybody else has a reason to require the upgrade. Apparently there was something else that snuck in here that I need to submit separately.

**Comment by gravyboat** [comment]
@iggy If you just want to change that I'll merge it. My other comment was just a 'nice to have'.

**Comment by iggy** [comment]
We don't need this anymore now that the latest released version has the -batch switch

**Pull request closed without merging**